src/components/Element/Article.jsx

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import clsx from 'clsx';
+import { useState, useEffect, useRef } from 'react';
+import { Calendar, Tip, Content } from './ArticleElement';
+
+export function Article() {
+  const [article, setArticle] = useState([]);
+  const [sentence, setSentence] = useState({});
+  const [position, setPosition] = useState({});
+  const [link, setLink] = useState('');
+  const markRef = useRef(null);
+
+  useEffect(() => {
+    axios
```

Comment:
can you refactor this function with async await and separate the logic between getting the data and how it is handled when the data has been retrieved.